src/components/splash/Splash.js

```diff
@@ -1,21 +1,27 @@
 import React from 'react'
-import { StyleSheet, View } from 'react-native'
+import { ImageBackground, StyleSheet } from 'react-native'
 import AnimationsLogo from '../common/animations/Logo'
-import wavePattern from '../../assets/wave50.svg'
+import wavePattern from '../../assets/splashWaves.svg'
 import Wrapper from '../common/layout/Wrapper'
 import Section from '../common/layout/Section'
 import Config from '../../config/config'
 
 const Splash = ({ animation }) => (
   <Wrapper style={styles.wrapper}>
     <Section style={styles.container}>
-      <View style={styles.backgroundWaves} />
-      <Section.Stack style={styles.content} grow justifyContent="center">
-        <AnimationsLogo animation={animation} />
-        <Section.Text fontSize={22} color="darkBlue">
-          {`V${Config.version}`}
-        </Section.Text>
-      </Section.Stack>
+      <ImageBackground
+        source={wavePattern}
+        imageStyle={styles.waves}
+        style={styles.backgroundWaves}
+        resizeMode="repeat"
+      >
+        <Section.Stack style={styles.content} grow justifyContent="center">
+          <AnimationsLogo animation={animation} />
+          <Section.Text fontSize={22} color="darkBlue">
+            V${Config.version}
```

Comment:
remove '$', as it's not an string literal anymore. It will come to the output

just V{Config.version}

Reply:
@serdiukov-o-nordwhale 
Oops, missed that.
Done see new commit